com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/BaseMultiInstanceTest.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace MLAPI.RuntimeTests
+{
+    public class BaseMultiInstanceTest
```

Comment:
I think to the theme before, this would be a better fit in its own PR

Reply:
moved to https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/914 and applied this to all other tests